### DISCUSSION TITLE
Issue regrading plant files in icuas24_folder

### DISCUSSION THREAD

**Discussion by Ro0412** [larics/icuas24_competition#29] [Q&A]
Are all the plant file png supposed to be empty plants in tomato pepper and eggplant folders?

**Comment by fpetric** [comment]
png files in model folders are plant textures that is overlayed on the "plant box" shape as textures and they are without fruit, yes. The fruits are 3D shapes of single color. If you open the .dae file in Blender or other 3D modelling software, you should be able to see the whole plant+fruit model.
